src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -305,4 +309,30 @@ public static String generateRandomEmail() {
         return String.format(EMAIL_TEMPLATE, randomAlphanumeric(10));
     }
 
+    @Override
+    public void afterTestClass(TestContext testContext) {
+        deleteTestCaseData();
+    }
+
+    private void deleteTestCaseData() {
+        if ("aat".equalsIgnoreCase(getenv("execution_environment"))) {
+            log.info("Delete test data script execution started");
+            try {
+                Connection connection = dataSource.getConnection();
```

Comment:
is this method executes once in the test run or for  each test class?

Reply:
once per suit